jax/experimental/jax2tf/saved_model_lib.py

```diff
@@ -101,17 +87,17 @@ def jax_fn_for_save_model(params, inputs):
       raise ValueError("For shape-polymorphic conversion a single "
                        "input_signature is supported.")
   tf_fn = jax2tf.convert(
-      jax_fn,
-      with_gradient=with_gradient,
-      in_shapes=[None, shape_polymorphic_input_spec],
-      enable_xla=enable_xla)
+    jax_fn,
+    with_gradient=with_gradient,
+    in_shapes=[None, shape_polymorphic_input_spec],
+    enable_xla=enable_xla)
 
   # Create tf.Variables for the parameters.
   param_vars = tf.nest.map_structure(
```

Comment:
Since we're using nest, I'd suggest using `map_structure_with_path` (I'm not sure if this is exposed in TF2 so you might need to use `import tensorflow.compat.v1 as tf1` or `dm-tree`) here and including a name for the variables based on their path. Could be a follow up, but otherwise the default names of `Variable_{1..N}` will not be super useful.

Reply:
`map_structure_with_path` is not exported as part of `tf.nest` in TF2. I just added a comment that one can use `dm-tree`